src/codegeneration/ClassTransformer.js

```diff
@@ -272,7 +301,9 @@ export class ClassTransformer extends TempVarTransformer{
     if (tree.name)
       name = tree.name.identifierToken;
     else
-      name = id(this.getTempIdentifier());
+      name = createIdentifierToken(this.getTempIdentifier());
+
+    var internalName = id(`${name}`);
```

Comment:
This was confusing to me as `name` was sometime `identifier` (if named, taken from `tree.name.identifier`) and sometimes `IDENTIFIER_EXPRESSION` (when created by `id()`). This makes it similar to `transformClassDeclaration` code.

Please let me know if you want me to change this differently.